src/ExpandableListItem/ExpandableListItem.js

```diff
@@ -373,6 +420,27 @@ module.exports = kind(
 		return true;
 	},
 
+	/**
+	* @private
+	*/
+	backKeyHandler: function () {
+		var current = Spotlight.getCurrent();
+
+		this.hasHistoryEntry = false;
+
+		// In the case where Spotlight focus is not on one of the items in the expandable, but there
+		// was still an entry history from this control, we must be in a situation where the pointer
+		// has moved away from the control and we have yet to spot another item. We should then
+		// effectively "pass on" the back action by calling the "pop" method of History.
+		if (current && current.isDescendantOf(this)) {
+			this.closeDrawerAndHighlightHeader();
+		} else {
+			EnyoHistory.pop();
```

Comment:
No, the default behavior of `HistorySupport` will cause the control to hide if we don't indicate we have handled the event.